src/libcadet/model/MultiChannelTransportModel.cpp

```diff
@@ -846,6 +873,7 @@ void MultiChannelTransportModel::multiplyWithDerivativeJacobian(const Simulation
 
 void MultiChannelTransportModel::setExternalFunctions(IExternalFunction** extFuns, unsigned int size)
 {
+	// Add exchange here
```

Comment:
add a "@todo" to that comment so that all open ends are easiliy found in the future